src/test/java/com/github/stickerifier/stickerify/media/MediaHelperTest.java

```diff
@@ -215,6 +215,14 @@ void convertGifVideo() throws Exception {
 		assertVideoConsistency(result, 512, 274, 10F, 1_000L);
 	}
 
+	@Test
+	void convertAviVideo() throws Exception {
+		var aviVideo = loadResource("valid.avi");
+		var result = MediaHelper.convert(aviVideo);
+
+		assertVideoConsistency(result, 512, 512, 30F, 3_000L);
+	}
```

Comment:
The expected outcome is a square when the original file is a square, as in this case.
If you inspect the file `valid.avi`, you'll notice it has both height and width of 511 pixels. They would get resized to 512 and it would still be a square after the conversion.